packages/runner-ct/src/lib/state.ts

```diff
@@ -285,7 +285,7 @@ export default class State {
 
   @action
   initializePlugins = (config: Cypress.RuntimeConfigOptions, rootElement: HTMLElement) => {
-    if (config.env.reactDevtools) {
+    if (config.env.reactDevtools && !config.isTextTerminal) {
```

Comment:
this will be inconvenient next time you need to debug a failure that occurs only in run-mode, but if this is the fix you need, then i get it